parsers/go.c

```diff
@@ -692,6 +672,7 @@ static void parseConstTypeVar (tokenInfo *const token, goKind kind)
 	do
 	{
 		tokenInfo *typeToken = NULL;
+		scope = scope_original;
```

Comment:
I think re-setting the scope variable here every loop and setting it to a different value below makes the code a bit confusing. I would suggest keeping the contents of "scope" without change and introduce new variable such as "member_scope" whose value is set in makeTag() below and pass this one to parseStructMembers().